workers/generic-worker/aws_helper_test.go

```diff
@@ -1,29 +1,25 @@
 package main
```

Comment:
I think we should remove the `--configure-for-aws`/`gcp`/`azure` functionality.  For Azure it doesn't work anymore anyway (customdata is no longer in the instance metadata), and we may have broken the others as well. The supported way to run generic-worker in a cloud is with worker-runner, although it's still supported to run generic-worker *without* worker-runner in a standalone fashion.

It's already updated in this PR so we can just delete the code it in a followup, but if in revising this PR the configure-for-cloud functionality gives you any trouble, I'd suggest preferring to remove rather than fix.

Reply:
Let's hold off deleting it until we have the new deploymentId support in worker-runner. At that point, I think we'll have feature parity.